README.md

```diff
@@ -28,10 +28,10 @@ pip install .
 
 ### NodeStorage
 
-```
->> from libsonata import NodeStorage
+```python
+>> import libsonata
```

Comment:
What is the purpose of the `>>` in the python code block ?

Reply:
to show the difference between what is input, and what is output